### DISCUSSION TITLE
[Help] how to get the calendar value

### DISCUSSION THREAD

**Discussion by sdnd2000** [fomantic/Fomantic-UI#1525] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
# Help Wanted
Hi, I am trying to the calendar selected value through Jquery, but I got the error "underfined", it is quick a basic operation, please advise. 

# Test case, 
Pleaes check it in JS fiddle. 
https://jsfiddle.net/1o2pzsk0/1/

**Comment by ko2in** [comment, marked as answer]
You have to use `calendar('get date')` to get the selected date (https://fomantic-ui.com/modules/calendar.html#/usage).

If you want to use jQuery `val()`, you have to select from the actual input field, not the DOM element that invoke the `calendar` module.

Example with `calendar('get date')` method (https://jsfiddle.net/nqb0zvk2/).

Example with jQuery `val()` method (https://jsfiddle.net/nqb0zvk2/1/).

**Comment by lubber-de** [comment]
The value is also available as a parameter for the `onSelect`, `onBeforeChange` and `onChange` callbacks
See https://jsfiddle.net/lubber/w07qu4vj/4/
